src/__tests__/OutlineButton.js

```diff
@@ -8,4 +8,20 @@ describe('OutlineButton', () => {
     const json = renderer.create(<OutlineButton theme={theme} />).toJSON()
     expect(json).toMatchSnapshot()
   })
+
+  test('disabled prop sets', () => {
+    const json = renderer.create(<OutlineButton disabled theme={theme} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('background-color', 'transparent')
+    expect(json).toHaveStyleRule('border-color', theme.colors.blue)
+    expect(json).toHaveStyleRule('color', theme.colors.blue)
+  })
+
+  test('without disabled prop sets', () => {
+    const json = renderer.create(<OutlineButton theme={theme} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('border-color', theme.colors.darkBlue, {
```

Comment:
@Degron I see `OutlineButton` has more styles on hover like `color`. Let's check for that here too

Reply:
Fixed it.